packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@vtex/store-ui'
+import React, { FC } from 'react'
+
+type Props = {
+  variant?: string
+  as: React.ElementType
+  width?: number
+  height?: number
+  src: string
+  alt: string
+  loading?: string
+}
+
+export const HeaderMinicartDrawerContentImage: FC<Props> = ({
+  loading = 'lazy',
+  ...props
+}) => {
+  const imageProps = {
+    loading,
+    ...props,
+  }
+
+  return (
+    <Box variant={props.variant}>
+      <Box {...imageProps} />
```

Comment:
This way we create one object less
```suggestion
      <Box loading={loading} {...props} />
```

Reply:
loading attribute does not exist on the box component, with object he infer the props. Do you have an idea how we fix it?

Reply:
<img width="627" alt="Screen Shot 2020-11-19 at 12 22 33" src="https://user-images.githubusercontent.com/10627086/99686004-ebba9500-2a61-11eb-9006-dcf634100e01.png">

Reply:
Oops, ok